internal/service/rbac/rbac.go

```diff
@@ -0,0 +1,90 @@
+package rbac
+
+import (
+	"context"
+
+	"github.com/eurofurence/reg-room-service/internal/config"
+	"github.com/eurofurence/reg-room-service/internal/web/common"
+)
+
+type CtxKeyValidator struct{}
+
+type Validator struct {
```

Comment:
I think Validator should be an interface so we can have a mock Validator in the context for unit tests.

Love the idea of just putting the Validator in the context!